pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go

```diff
@@ -33,8 +33,6 @@ const (
 	RequestedToCapacityRatioName = "RequestedToCapacityRatio"
 	minUtilization               = 0
 	maxUtilization               = 100
```

Comment:
Are these 2 above still used?

Reply:
`minUtilization` isn't . `maxUtilization` is still used when building [the scoring function](https://github.com/kubernetes/kubernetes/blob/c37933e07df5a716a30452ace6e23d44d5ea4eeb/pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go#L131).

I'll remove the former and see if the latter is really needed in the following PR.

Reply:
sg